tests/cases/osmLayer.js

```diff
@@ -1,6 +1,9 @@
 // Test geo.core.osmLayer
 var geo = require('../test-utils').geo;
 var $ = require('jquery');
+var mockAnimationFrame = require('../test-utils').mockAnimationFrame;
```

Comment:
I moved the rendering into a requestAnimationFrame callback for d3 features. Before, as we added tiles, we would render them as they came in,  This means that when you panned and we loaded multiple tiles, the tiles would render many times in that animation frame, which caused a huge performance lag for that moment.  By using an animation frame, we render all of the new tiles once, and it limits the performance hit.  It still takes longer than I'd like to add many svg elements at once, but it is swifter than it was.